book/layout.md

```diff
@@ -213,31 +225,49 @@ INLINE_ELEMENTS = [
 ]
 ```
 
-A block layout object looks at its children's tags to determine
-whether it should have block or inline contents:
+To determine whether an element has an `InlineLayout` or a
+`BlockLayout` we compare its children against that list. Let's add a
+top-level `layout_mode` function that function looks through the
+children of a node and categorizes them as either "text", including
+formatting tags, or containers:
```

Comment:
Oh. Is "containers" the terminology you're using for block elements? Is that a good name to use?

Reply:
It's not. Do you have better ideas? Here's what I'm trying to do—I want to define what "block" and "inline" layout modes are for, and if I say "block layout mode is for block elements" it'll be confusing why this is even a question... "Containers" is pretty weird though, maybe it's best to talk about "blocks" and "text"?

Reply:
Ok just add a sentence defining "container" earlier in the chapter, in the conceptual section?